bahmni-docker/.env

```diff
@@ -47,8 +46,11 @@ OPENMRS_DB_CREATE_TABLES='false'
 OPENMRS_DB_AUTO_UPDATE='true'
 OPENMRS_MODULE_WEB_ADMIN='false'
 OPENMRS_DEBUG='true'
+OPENMRS_LOAD_DEMO_DATA='true'
+OPENMRS_DB_DUMP_GZIP_URL=''
 OPENMRS_UPLOAD_FILES_PATH=./openmrs-uploads
 BAHMNI_OPENMRS_MODULES_PATH=
+MYSQL_ROOT_USERNAME=root
```

Comment:
Is there a way we can keep Root username & password in CI/CD secrets instead of passing it using .env file.
Because this will be still visible in code repo, which will be a security risk.

Reply:
This is not going to deployment Mradul. This is for local development setup. This enables you to just clone and run bahmni locally.